### ISSUE TITLE
Update product names to IBM Cloud, not Bluemix

### ISSUE THREAD

**Issue by stevemart** [IBM/innovate-digital-bank#9]
Lots of instances of "Bluemix" should be replaced with IBM Cloud

Same with the `bx` command, it should be replaced with the new `ibmcloud` CLI.

**Comment by stevemart** [comment]
Hi @amalamine -- can you respond to this one.

**Comment by stevemart** [comment]
Note that the `bx` command is also used in the .travis file and is failing there with the following error:

```
> innovate@1.0.0 test /home/travis/build/IBM/innovate-digital-bank
> sudo bx dev build --debug
sudo: bx: command not found
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
```

**Comment by stevemart** [comment]
fixed in #12
